agent/agent-tooling/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/quickpulse/QuickPulse.java

```diff
@@ -77,7 +83,8 @@ private void initialize(
       Supplier<URL> endpointUrl,
       Supplier<String> instrumentationKey,
       @Nullable String roleName,
-      @Nullable String roleInstance) {
+      @Nullable String roleInstance,
+      boolean backCompatNonNormalizedCpuPercentage) {
```

Comment:
this is confusing.. backCompatNonNormalizedCpuPercentage is used normalized cpu percentage.. 
can you rename to usedNormalizedCpuPercent? and add the explaination for backward compatibility of non-normalized cpu percentage?